sounds/ammo_low.txt

```diff
@@ -1,3 +1,3 @@
 Derived alert.wav by Corsica_S
 http://freesound.org/people/Corsica_S/sounds/156453/
-http://creativecommons.org/licenses/by/3.0/
+http://creativecommons.org/licenses/by/4.0/
```

Comment:
This must be freesound updating their licenses, as I usually copy the CC link from the page when creating these files

Reply:
I don't think they update this automatically, as many sounds still use 3.0.  
I think the creators manually switched to 4.0 (maybe freesound is asking them to update?).